lv_objx/lv_label.c

```diff
@@ -158,31 +161,33 @@ void lv_label_set_text(lv_obj_t * label, const char * text)
     // For more information, please refer to the following references:
     // https://gridplusteam.slack.com/archives/GA6H5KR9U/p1614013623031400
     // https://github.com/GridPlus/k8x_firmware_production/pull/1887
-
     size_t textBufSz = strlen(text);
-    char processedText[textBufSz];
-    size_t j = 0; // lagging index
-    for(size_t i = 0; i < textBufSz; i++) {
-        if (text[i] < 0x7f) {
-            processedText[j] = text[i];
-        } else if ((textBufSz - i) >= 2) {
-            if (text[i] == 0xef && (text[i + 1] == 0xa0 || text[i + 1] == 0xa4)) {
-                processedText[j] = text[i];
-                i++; j++;
+    #define MAX_PARSEABLE_STRING_SIZE 400
```

Comment:
Why is this #define here and not at the top of the file like every other define?

Reply:
True. I could have made it an enum too, to keep it local. I didn't make it global (except I did with the `#define` because this is a workaround that only applies to the label objects, and not any other text-receiving object -- that said this code is going to be deprecated soon when the new UI work is done, so I am ok with this.